packages/ui/src/theme-web/colors.ts

```diff
@@ -0,0 +1,8 @@
+import { defineTokens } from '@pandacss/dev';
+
+// ts-unused-exports:disable-next-line
+export const colors = defineTokens.colors({
+  // TODO: Move to mono repo
+  current: { value: 'currentColor' },
+  overlay: { value: 'rgba(0,0,0,0.4)' },
```

Comment:
I think these can be removed bc in tokens pkg now?

Reply:
We have `overlay` in tokens but not `current`. I don't think that needs to be in `tokens` though?